docs/sdks/server-sdks/node/quickstart.mdx

```diff
@@ -0,0 +1,313 @@
+:::
+
+### Assign a variant
+
+After initializing the SDK with the `logAssignment()` function, you can make assignments in the same way as for feature flags.
+
+```js
+import * as EppoSdk from "@eppo/js-server-sdk";
+
+// Get SDK instance
+const eppoClient = EppoSdk.getInstance();
+
+// Define assignment parameters
+const featureKey = "my-neat-feature";
+const subjectKey = getCurrentUser() || "anonymous";
+const subjectAttributes = {
+    'country': user.country,
+    'device': user.device,
+};
+const defaultValue = "default-value";
+
+// Make an assignment
+const variation = eppoClient.getStringAssignment(
+  featureKey,
+  subjectKey,
+  subjectAttributes,
+  defaultValue
+);
+
+if (variation === "landing-page-a") {
+    return handleLandingPageA();
+} else if (variation === "landing-page-b") {
+    return handleLandingPageB();
+} else {
+    return handleLandingPageDefault();
+}
+```
+
+You can see the full steps for running an experiment including UI steps in the [experiment quick start](/experiment-allocation-quickstart/).
+
+
+## Contextual Bandits
+
+Contextual Multi-Armed Bandits are a way to dynamically optimize assignments based on user context. A bandit balances exploration of new actions with exploitation of known successful actions to maximize a specified metric.
+
+### Bandit Setup
+
+Setting up a bandit is very similar to setting up an experiment but with an additional logging function.
+
+```js
+// Import Eppo's logger interfaces and client initializer
+import { IAssignmentLogger, IBanditLogger, init } from "@eppo/node-server-sdk";
+
+// Define an assignment logger for recording variation assignments
+const assignmentLogger: IAssignmentLogger = {
+  logAssignment(assignment: IAssignmentEvent) {
+    console.log("TODO: save assignment information to data warehouse", assignment);
+  }
+};
+
+// Define a bandit logger for recording bandit action assignments
+const banditLogger: IBanditLogger = {
+  logBanditAction (banditEvent: IBanditEvent) {
+    console.log("TODO: save bandit action information to the data warehouse, ensuring the column names are as expected", banditEvent);
+  }
+};
+
+// Initialize the SDK with both loggers provided
+await init({
+  apiKey: "<SDK_KEY>",
+  assignmentLogger,
+  banditLogger
+});
+```
+
+:::note
+In a production application, you would want to replace the `console.log()` statement with an actual logging system. We have documentation on how to set up logging with multiple popular data warehouses and logging systems in the [Assignment page](/sdks/client-sdks/javascript/assignments/#logging-data-to-your-data-warehouse).
+:::
+
+### Query the bandit for actions
+
+Instead of making simple assignments with a bandit, you will want to query the bandit for actions. 
+
+```js
+import * as EppoSdk from "@eppo/js-server-sdk";
+import { Attributes, BanditActions } from "@eppo/js-client-sdk-common";
```

Comment:
Note that this example--unlike the others--is using TypeScript now. We should decide if we want our Node examples to use TypeScript or vanilla JS.

It looks like StatSig uses TypeScript. I imagine that is what most Node.js developers are using these days.